### DISCUSSION TITLE
FFmpeg path

### DISCUSSION THREAD

**Discussion by dodgerblue1** [Shabinder/SpotiFlyer#1052] [Q&A]
Where is the path? There is no official documentation on this(or I can't find it). Please help :)

**Comment by Shabinder** [comment]
Install FFmpeg and add it to your path.
See any blog/guide,
for reference : https://soundbound.app/ffmpeg-windows/

Edit: Updated Tutorial link with a simpler one.

**Reply by djbanana** [reply]
i tried following the instructions to the letter it still says no FFmeg found at path I went as far as to copy the ffmeg folder in the installation directory

**Reply by D4vidLuc4s** [reply]
Follow this https://phoenixnap.com/kb/ffmpeg-windows
That worked for me

**Reply by Shabinder** [reply]
or Either this: https://soundbound.app/ffmpeg-windows/
